docuware-connector/src/com/axonivy/connector/docuware/connector/DocuWareEndpointConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package com.axonivy.connector.docuware.connector;
+
+public class DocuWareEndpointConfiguration {
+
+	
+	private String username;
+	
+	private String password;
```

Comment:
secrets should not be held as String ... rather as byte[] to make heap-dump memory analysis harder. 
but hopefully we can get rid of this altogether, and prefer an authentication feature to deal with the user:pass

Reply:
removed it togheter with the unnecessary additional basic auth in the header params